htdocs/js/ui/scratchpad.js

```diff
@@ -55,6 +55,14 @@ RCloud.UI.scratchpad = (function() {
                 RCloud.UI.thumb_dialog.init();
 
                 $('#update-thumb').click(function() {
+
+                    // select the thumb in the assets:
+                    var thumb = shell.notebook.model.get_asset('thumb.png');
+
+                    if(thumb) {
+                        thumb.controller.select();
```

Comment:
This doesn't.. quite.. work, right? If you want to be a hero, you could figure out why this helps and do something better:

https://github.com/att/rcloud/blob/develop/htdocs/js/notebook/asset_view.js#L62-L64